datafusion/physical-expr/src/expressions/binary.rs

```diff
@@ -2609,14 +2441,14 @@ mod tests {
             Operator::Divide,
             create_decimal_array(
                 &[
-                    Some(99193548387), // 0.99193548387
```

Comment:
why these changes?

Reply:
Because the rules for precision of decimal division are now correct #6828

Reply:
I think that it the rules for precision of decimal division are changed. I won't say previous rules are incorrect.